source/_components/asuswrt.markdown

```diff
@@ -26,6 +26,25 @@ asuswrt:
   username: YOUR_ADMIN_USERNAME
 ```
 
+To enable ASUSWRT sensors as part of your installation, reference the following example configuration:
```

Comment:
Move this sample configuration to the end of the file and add a linkable title please.

Reply:
Done